examples/ex9.cpp

```diff
@@ -812,13 +816,26 @@ class FluxCorrectedTransport
       neighborDof.SetSize(ne*numDofs, numBdrs);
       
       double dist = 1. / double(p);
-      Vector velEval;
+      Vector velEval; // TODO rm
       
       for (k = 0; k < ne; k++)
       {
          ///////////////////////////
          // Element contributions //
          ///////////////////////////
+         /*for (m = 0; m < numSubcells; m++)
+         {
+            dofInd = numSubcells*k+m;
+            const FiniteElement *el0 = SubFes0.GetFE(dofInd);
+            const FiniteElement *el1 = SubFes1.GetFE(dofInd);
+            tr = ref_mesh->GetElementTransformation(dofInd);
+            //tr->Jacobian().Print();
```

Comment:
These transformations are wrong. Even with the SetCurvature calls. Can you help me fix it please?

Reply:
Can you clarify when is the transformations are correct? The following should work...
```c++
   int basis_lor = BasisType::GaussLobatto; // BasisType::ClosedUniform;
   Mesh *mesh_lor = new Mesh(mesh, order, basis_lor);
   ElementTransformation *tr = mesh_lor->GetElementTransformation(0);
   tr->SetIntPoint(&ip); // define ip
   tr->Jacobian().Print();
```